### ISSUE TITLE
[0.3.3] Error opening Rich Text Editor in Scenario

### ISSUE THREAD

**Issue by 0ptera** [LuziferSenpai/Rich-Text-Helper#8]
Thanks for the quick fix on loading a scenario, however opening the editor in transport madness results in this error:
```
Error MainLoop.cpp:1202: Exception at tick 292: The mod Rich Text Helper caused a non-recoverable error.
Please report this error to the mod author.

Error while running event Rich_Text_Helper::on_gui_click (ID 1)
__Rich_Text_Helper__/scripts/Colors.lua:53: attempt to index upvalue 'script_data' (a nil value)
stack traceback:
	__Rich_Text_Helper__/scripts/Colors.lua:53: in function 'ColorUpdate'
	__Rich_Text_Helper__/scripts/Main.lua:35: in function 'MainGUIToggle'
	__Rich_Text_Helper__/scripts/Main.lua:81: in function 'click'
	__Rich_Text_Helper__/scripts/Main.lua:149: in function 'handler'
	__core__/lualib/event_handler.lua:44: in function <__core__/lualib/event_handler.lua:42>
```

**Comment by LuziferSenpai** [comment]
Yea, on that I currently work on. Its a shitty mess! You can see that in the mod-making Channel.

**Comment by LuziferSenpai** [comment]
Fixed in 0.3.4
